Sources/Host/Linux/PTrace.cpp

```diff
@@ -316,6 +318,15 @@ ErrorCode PTrace::writeRegisterSet(ProcessThreadId const &ptid, int regSetCode,
 
   return kSuccess;
 }
+
+#if defined(ARCH_X86) || defined(ARCH_X86_64)
+size_t PTrace::computeDebugRegOffset(int idx) {
```

Comment:
Also, keeping this as a static private function in the implementation and not adding it to the interface of the class sounds like a better idea.